gpu/people/tools/people_tracking.cpp

```diff
@@ -32,6 +32,13 @@ class PeopleTrackingApp
   public:
     PeopleTrackingApp () : viewer ("PCL People Tracking App") {}
 
+    ~PeopleTrackingApp ()
+    {
+      if (m_proc) {
```

Comment:
Use `unique_ptr` for m_proc instead. You're using `new` to create m_proc but `free` to delete. That's a big issue. It's tools so this shouldn't be a breaking change to modify the interface to accept a unique_ptr in ctor.